index.js

```diff
@@ -0,0 +1,22 @@
+const axios = require("axios").default;
+const player = require('play-sound')();
+const api = axios.get(
```

Comment:
Podrias usar una constante mejor para el endpoint .  Lo correcto seria usar una variable de `env` y si no esta presente lo hace como default el string que tienes.

```suggestion

const HEB_ENDPOINT = process.env.HEB_ENDPOINT || ' http:/ ....' 

const api = axios.get(HEB_ENDPOINT);
```